CIAC-12287/XSIAM-PB-Suspicious-msiexec-execution (#37859)
demisto/content: Packs/Core/Playbooks/playbook-Suspicious_msiexec_execution_README.md

This playbook addresses the following alerts:
 
- Msiexec execution of an executable from an uncommon remote location with a specific port
- Msiexec execution of an executable from an uncommon remote location without properties
 
Playbook Stages:
  
Analysis: 
 
- Check extracted domain's reputation
- Check extracted domain's prevalence
 
 
Investigation:
 
- Check for suspicious related alerts: 
    - Local Analysis Malware
    - Mitre Techniques:
        - T1140 - Deobfuscate/Decode Files or Information
        - T1059 - Command and Scripting Interpreter 

Containment:
 
- Kill causality process

> - Block maliciou URL

Reviewer: ```suggestion
- Block malicious URL
```

 
Requirements: 
 

> For any response action, you need one of the following integrations:

Reviewer: ```suggestion
For any response action, you need the following integration:
```

> - PAN-OS.

Reviewer: ```suggestion
PAN-OS
```

## Dependencies

This playbook uses the following sub-playbooks, integrations, and scripts.

### Sub-playbooks

> * PAN-OS - Block URL - Custom URL Category
> 
> ### Integrations
> 
> * CortexCoreIR

Reviewer: ```suggestion
PAN-OS - Block URL - Custom URL Category

### Integrations

CortexCoreIR
```
